app/src/org/commcare/android/view/EntityView.java

```diff
@@ -530,5 +491,8 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
                 views[i].setLayoutParams(params);
             }
         }
-    }*/
+        
+        // Use children's new widths
+        super.onMeasure(widthMeasureSpec, heightMeasureSpec);

```

Comment:
Can you explain why there are two calls to super.onMeasure in this method?
Could you remove the first one or the second one or are they both necessary?